source/databricks/calculation_engine/package/calculation/wholesale/get_metering_points_and_child_metering_points.py

```diff
@@ -35,24 +35,35 @@ def get_metering_points_and_child_metering_points(
 
     es = "energy_supplier_id_temp"
     mp = "metering_point_id_temp"
+    fd = "from_date_temp"
+    td = "to_date_temp"
```

Comment:
Can we lower the cognitive load by not introducing uncommon abbrevations?
```suggestion
    from_date = "from_date_temp"
    to_date = "to_date_temp"
```